### ISSUE TITLE
[Feature Request] tray icon search box

### ISSUE THREAD

**Issue by benwbooth** [keeweb/keeweb#320]
It's useful to have a tray icon for easily selecting a password entry for quick copy-paste without having to bring up the main keeweb window. This is something I use a lot in Enpass, and find it really convenient.

Basically, clicking on the tray icon should bring up a small search-as-you-type combo box menu. If the database is locked, then the usual unlock screen can be displayed in the small window. Once the password is entered, the search box reappears. It would be great if there was a button for copying the password in one click, and another for the username/email. A third button could bring up the full record. Keyboard shortcuts for copying the username/password and/or autotype integration would be useful as well.

**Comment by antelle** [comment]
That would be a good addition 👍 
But it's not supported in Electron, so we need to hack on native API and create an app. Or maybe a sort of popup form which appears when you click this icon. Needs investigation, for now.

**Comment by benwbooth** [comment]
I found [this](https://github.com/electron/electron/blob/master/docs/api/tray.md) on the electron github page. Would it be possible to display a small window near the tray icon or the mouse pointer on a click event? Or are electron apps limited to single window applications?

**Comment by antelle** [comment]
Yes, we are using this module to display tray icon and menu. But you're right, we could display an additional window there, e.g. dropbox does this (this window should be styled depending on os though but it's also can be implemented).

**Comment by antelle** [comment]
So, now there are more options to implement it and it's easier to implement.
